<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoFormatFieldWidget</class>
 <widget class="QWidget" name="VideoFormatFieldWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bitrate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Other opts</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Codec</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Pixel format</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="spinBox_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_3"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>textEdited(QString)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>VideoFormatFieldWidget</receiver>
   <slot>handleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleChanged()</slot>
 </slots>
</ui>
